CMakeLists.txt

```diff
@@ -1,5 +1,10 @@
 cmake_minimum_required(VERSION 3.3)
-project(opensmt VERSION 2.0.0 LANGUAGES CXX)
+cmake_policy(VERSION 3.12)
```

Comment:
At least in the cluster the cmake complains something without this line.

Reply:
I think we have to be more careful and setting options per policy, and not globally.
On machine with older version of CMake it gives this error:
```
CMake Error at CMakeLists.txt:2 (cmake_policy):
 An attempt was made to set the policy version of CMake to "3.12" which is
 greater than this version of CMake.  This is not allowed because the
 greater version may have new policies not known to this CMake.  You may
 need a newer CMake version to build this project.
```

Reply:
Ok.